We haven't used Travis in a very long time.

ltratt opened the pull request:
It also turns out that our previous pull requests hadn't passed through CI properly due to an incorrect GitHub setting (mea culpa!). This PR does do something useful, and will also make sure we pass CI before a release!

ltratt committed:
We haven't used Travis in a very long time.

ratmice commented:
I think the issue is `cargo deny check`, I had to remove `unlicensed = deny` (now the default according to [this](https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html), and change the unicode license to Unicode-3.0,
keeping `Unicode-DFS-2016` produced a warning

```
diff --git a/deny.toml b/deny.toml
index 31719a9..5f47cf2 100644
--- a/deny.toml
+++ b/deny.toml
@@ -1,8 +1,7 @@
 [licenses]
-unlicensed = "deny"
 confidence-threshold = 1.0
 allow = [
     "Apache-2.0",
     "MIT",
-    "Unicode-DFS-2016"
+    "Unicode-3.0",
 ]
```

ltratt committed:
Update `deny.toml`.

We need to remove the deprecated `unlicensed` field and reflect a change
in `unicode-ident`'s license.

ltratt commented:
Yep, I accidentally recreated the same changes you did (I should have checked first!). Ready for merging (I don't think that second commit should be squashed).
